  <launch>
    <!-- Turtlesim Node-->
    <node pkg="turtlesim" type="turtlesim_node" name="sim"/>

    <node pkg="turtlesim" type="turtle_teleop_key" name="teleop" output="screen"/>
    <!-- Axes -->
    <param name="scale_linear" value="2" type="double"/>
    <param name="scale_angular" value="2" type="double"/>


<!-- see http://wiki.ros.org/roslaunch/XML/node -->
    <node pkg="pkg_name_jeff_tf_learn" type="any_name_ok_launch_name"
          args="/turtle1" name="node_name1_my_tf_broadcaster" />
    <node pkg="pkg_name_jeff_tf_learn" type="any_name_ok_launch_name"
          args="/turtle2" name="node_name2132_my_tf_broadcaster" />
 
    <node pkg="pkg_name_jeff_tf_learn" type = "jeff_tf_listener"
          name = "cpp_file_node_name_tf_listener" />

    <node pkg="pkg_name_jeff_tf_learn" type = "create_new_frame"
          name = "create_new_frame_name_in_lanuch_file" />

  </launch>
